types/epub-gen/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for epub-gen 0.1
+// Project: https://github.com/cyrilis/epub-gen
+// Definitions by: Sam Tyler <https://github.com/styler3>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Chapter {
```

Comment:
I _believe_ types won't be available on the export anymore, now that you've done `export = Epub`.

You likely (?) want to combine this with a `namespace` to merge onto `Epub` so you can reference it. I think there are some examples of this in the repo if you do a little searching.

Reply:
Sorry @jakebailey, I've never worked with either CommonJS nor used `namespace` before so I don't really understand what I'm doing here. It did seem to work for me the way it was before :shrug:.

I've updated it basing it on `types/11ty__eleventy-img/generate-html.d.ts` as an example.

Is that right?

Reply:
I think what you've done looks correct now, yeah.